geopyspark-backend/geotrellis/src/main/scala/geopyspark/geotrellis/RasterRDD.scala

```diff
@@ -123,8 +123,16 @@ abstract class TileRDD[K: ClassTag] {
     }
   }
 
+  def toInt: TileRDD[_] =
+    toInt(rdd.map { x => (x._1, x._2.convert(IntCellType)) })
+
+  def toDouble: TileRDD[_] =
+    toDouble(rdd.map { x => (x._1, x._2.convert(DoubleCellType)) })
```

Comment:
These aren't enough unfortunately. Aside from converting between floating and integer values you may also need to control the width of your data types in order to save storage/memory.
So we need to mirror this function: https://github.com/locationtech/geotrellis/blob/master/spark/src/main/scala/geotrellis/spark/MultibandTileLayerRDDMethods.scala#L28

Thankfully there is a mapping from String => CellType: https://github.com/locationtech/geotrellis/blob/master/raster/src/main/scala/geotrellis/raster/CellType.scala#L412